editor/js/Script.js

```diff
@@ -429,6 +429,26 @@ function Script( editor ) {
 
 	} );
 
+	signals.objectChanged.add( function ( object ) {
+
+		if ( editor.scripts[ object.uuid ].includes( currentScript ) ) {
+
+			signals.editScript.dispatch( currentObject, currentScript );
+
+		}
+
+	} );
+
+	signals.scriptChanged.add( function ( script ) {
+
+		if ( script === currentScript ) {
+
+			signals.editScript.dispatch( currentObject, currentScript );
+
+		}
+
+	} );
```

Comment:
Instead of dispatching `editScript` signal, both handlers can be written like so:
```js
signals.objectChanged.add( function ( object ) {

	if ( object === currentObject ) {

		title.setValue( currentObject.name + ' / ' + currentScript.name );

	}

} );

signals.scriptChanged.add( function ( script ) {

	if ( script === currentScript ) {

		title.setValue( currentObject.name + ' / ' + currentScript.name );

	}

} );
```

Reply:
GOOD!